telemetry/src/main/java/org/mozilla/telemetry/Telemetry.java

```diff
@@ -83,13 +84,27 @@ public Telemetry queueEvent(final TelemetryEvent event) {
         executor.submit(new Runnable() {
             @Override
             public void run() {
-                EventsMeasurement measurement = ((TelemetryEventPingBuilder) pingBuilders.get(TelemetryEventPingBuilder.TYPE))
-                        .getEventsMeasurement();
+                // We migrated from focus-event to mobile-event and unfortunately, this code was hard-coded to expect
+                // a focus-event ping builder. We work around this by checking our new hardcoded code first for the new
+                // ping type and then falling back on the legacy ping type.
+                final TelemetryPingBuilder mobileEventBuilder = pingBuilders.get(TelemetryMobileEventPingBuilder.TYPE);
+                final TelemetryPingBuilder focusEventBuilder = pingBuilders.get(TelemetryEventPingBuilder.TYPE);
```

Comment:
We should probably rename `TelemetryEventPingBuilder` to `TelemetryFocusEventPingBuilder` for clarity.

Reply:
We spoke on Slack: this is used by both Focus and Rocket so we don't want to make the breaking change to rename this.